src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -0,0 +1,266 @@
+                  >
+                    <EuiText size="s">
+                      {i18n.translate('visualizations.newVisWizard.exploreOptionLinkText', {
+                        defaultMessage: 'Explore Options',
+                      })}{' '}
+                      <EuiIcon type="sortRight" />
+                    </EuiText>
+                  </EuiLink>
+                </EuiCard>
+              </EuiFlexItem>
+            )}
+            {props.visTypesRegistry.getByGroup(VisGroups.TOOLS).length > 0 && (
+              <EuiFlexItem className="visNewVisDialog__toolsCard">
+                <EuiSpacer size="m" />
+                <EuiTitle size="xs">
+                  <span data-test-subj="visGroup-tools">
+                    {i18n.translate('visualizations.newVisWizard.toolsGroupTitle', {
+                      defaultMessage: 'Tools',
+                    })}
+                  </span>
+                </EuiTitle>
+                <EuiSpacer size="s" />
+                <div className="visNewVisDialog__toolsCardGroupContainer">
+                  {props.visTypesRegistry.getByGroup(VisGroups.TOOLS).map((visType) => (
+                    <ToolsGroup
+                      visType={visType}
+                      key={visType.name}
+                      onVisTypeSelected={props.onVisTypeSelected}
+                      showExperimental={props.showExperimental}
+                    />
+                  ))}
+                </div>
+              </EuiFlexItem>
+            )}
+          </EuiFlexGrid>
+          <EuiSpacer size="m" />
+          <EuiDescriptionList
+            className="visNewVisDialogGroupSelection__footerDescriptionList"
+            type="responsiveColumn"
+          >
+            <EuiDescriptionListTitle className="visNewVisDialogGroupSelection__footerDescriptionListTitle">
+              <FormattedMessage
+                id="visualizations.newVisWizard.learnMoreText"
+                defaultMessage="Want to learn more?"
+              />
+            </EuiDescriptionListTitle>
+            <EuiDescriptionListDescription>
+              <EuiLink href={visualizeGuideLink} target="_blank" external>
+                <FormattedMessage
+                  id="visualizations.newVisWizard.readDocumentationLink"
+                  defaultMessage="Read documentation"
+                />
+              </EuiLink>
+            </EuiDescriptionListDescription>
+          </EuiDescriptionList>
+        </div>
+      </EuiModalBody>
+    </>
+  );
+}
+
+const VisGroup = ({ visType, onVisTypeSelected }: VisCardProps) => {
+  const onClick = () => onVisTypeSelected(visType);
+  return (
+    <EuiFlexItem>
+      <EuiCard
+        titleSize="xs"
+        title={<span data-test-subj="visTypeTitle">{visType.title}</span>}
+        onClick={onClick}
+        isDisabled={isVisTypeAlias(visType) && visType.disabled}
+        betaBadgeLabel={
+          isVisTypeAlias(visType) && visType.disabled
```

Comment:
nit: please move` isVisTypeAlias(visType) && visType.disabled` into constant and use it for `betaBadgeLabel` and `betaBadgeTooltipContent`
we don't need to calculate that value twice